rules/linux/lnx_install_root_certificate.yml

```diff
@@ -0,0 +1,20 @@
+title: Install Root Certificate
+id: 78a80655-a51e-4669-bc6b-e9d206a462ee
+description: Detects installed new certificate
+references:
+    - https://attack.mitre.org/techniques/T1553/004/
+author: Ömer Günal
```

Comment:
Hello @omergunal !

Thank you for your contribution.
Would you mind adding "oscd.community" to the author field?
It will help with the promotion of the initiative and engage more people in the next sprints.